001-cron.php

```diff
@@ -79,6 +79,30 @@ function wpcom_vip_disable_jetpack_sync_on_cron_shutdown( $load_sync ) {
 }
 
 /**
+ * Log details of fatal error in callback that Cron Control caught
+ *
+ * @param $event object
+ * @param $error \Throwable
+ */
+function wpcom_vip_log_cron_control_event_for_caught_error( $event, $error ) {
+	$message = sprintf( 'PHP Fatal error:  Caught Error: %1$s in %2$s:%3$d', $error->getMessage(), $error->getFile(), $error->getLine() );
+	error_log( $message );
+
+	wpcom_vip_log_event_object( $event, 'Caught' );
```

Comment:
Since we separate fatals from other log entries, I hoped this would group the errors together to simplify correlating the error with its extra data. That's not happening right now (our custom "fatals" are ending up in the regular log), and if it won't work that way, I'm happy to change the prefix.